nautobot/extras/management/commands/fix_custom_fields.py

```diff
@@ -0,0 +1,40 @@
+from logging import getLogger
+
+from django.contrib.contenttypes.models import ContentType
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from nautobot.extras.models import CustomField
+from nautobot.extras.utils import FeatureQuery
+
+logger = getLogger("nautobot.extras.fix_custom_fields")
+
+
+class Command(BaseCommand):
+    help = "Reset custom field attachments"
+
+    def handle(self, *args, **kwargs):
+        """Run through all objects and ensure they are associated with the correct custom fields."""
+        content_types = ContentType.objects.filter(FeatureQuery("custom_fields").get_query())
+
+        for content_type in content_types:
+            logger.info(f"Processing ContentType {content_type}")
+            model = content_type.model_class()
+            custom_fields_for_content_type = content_type.custom_fields.all()
+            custom_field_names_for_content_type = [cf.name for cf in custom_fields_for_content_type]
+            with transaction.atomic():
+                for obj in model.objects.all():
+                    obj_changed = False
+                    # Provision CustomFields that are not associated with the object
+                    for custom_field in custom_fields_for_content_type:
+                        if not obj._custom_field_data[custom_field.name]:
+                            logger.debug(f"Adding CustomField {custom_field.name} to {obj}")
```

Comment:
would it make sense to use `logger.info` instead, or `self.stdout.write('')` ? 
if the log level is not properly configured nothing will be visible when we execute this command